tests/ui/target-feature/issue-108680-1.stderr

```diff
@@ -0,0 +1,12 @@
+error[E0133]: call to function with `#[target_feature]` is unsafe and requires unsafe function or block
+  --> $DIR/issue-108680-1.rs:10:5
+   |
+LL |     foo();
+   |     ^^^^^ call to function with `#[target_feature]`
+   |
+   = help: in order for the call to be safe, the context requires the following additional target features: neon aes sha2 sha3 sm4.
```

Comment:
ideally we'd backtick and separate them with commas + and, e.g.: `neon`, `aes`, `sha2`, `sha3`, and `sm4`.

Reply:
I'd prefer it emit like \`+neon,+aes,+sha2,+sha3,+sm4\`. Or to use a **string**, thus "+neon,+aes,+sha2,+sha3,+sm4", since these arguments are strings at the end of the day.

Reply:
So, for now I implemented the @compiler-errors suggestion, but my question here is for @workingjubilee, the `+` isn't for the target features enabled in the build config? Because the `help` is only showing the ones that aren't enabled in the `#[target_feature]`, the `+` sign could be confusing.

Reply:
Oh that's a good point. Nevermind then!